kafka-ui-react-app/src/components/Topics/New/__test__/New.spec.tsx

```diff
@@ -79,12 +79,12 @@ describe('New', () => {
       userEvent.click(screen.getByText(/submit/i));
     });
 
-    await waitFor(() =>
-      expect(mockedHistory.location.pathname).toBe(
-        clusterTopicPath(clusterName, topicName)
-      )
-    );
-    expect(mockedHistory.push).toBeCalledTimes(1);
-    expect(createTopicAPIPathMock.called()).toBeTruthy();
+    await waitFor(() => {
+      return expect(mockedHistory.location.pathname).toBe(
+        `${clusterTopicPath(clusterName, topicName)}`
+      );
+    });
+    expect(mockedHistory.push).toBeCalledTimes(0);
```

Comment:
is it correct behaviour?

Reply:
I removed extra string wrapper, if that's what you mean

Reply:
You have changed `toBeCalledTimes` from 1 to 0
`expect(createTopicAPIPathMock.called())` was `true` now it is `false`